src/modules/users/CheckPWQuality.h

```diff
@@ -38,10 +38,12 @@ class PasswordCheck
     using AcceptFunc = std::function<bool( const QString& )>;
     using MessageFunc = std::function<QString()>;
 
+    enum CheckId { DEFAULT, LIBPWQ };
```

Comment:
Needs dox, and I'm not a big fan of these names; perhaps InternalCheck and LibPQWCheck, and rename the *CheckId* (since it's **not** an indentity) to *CheckFamily* to emphasise that it applies to a family of checks?